backend/app/rest/api/rss.go

```diff
@@ -56,7 +56,6 @@ func (s *rss) postCommentsCtrl(w http.ResponseWriter, r *http.Request) {
 
 	w.Header().Set("Content-Type", "application/xml; charset=utf-8")
 	w.WriteHeader(http.StatusOK)
-
```

Comment:
This is just for consistency. I've checked all `WriteHeader(http.StatusOK)` for the bug's presence, and that is the only place in the `rss.go` file with that extra newline, while two others don't.